spectrocloud/resource_application_profile.go

```diff
@@ -113,6 +112,12 @@ func resourceApplicationProfile() *schema.Resource {
 								Type: schema.TypeString,
 							},
 						},
+						"install_order": {
+							Type:        schema.TypeInt,
+							Description: "The install orders will line up installation ascending, based on priority values set on multiple tiers",
```

Comment:
```suggestion
							Description: "The installation order priority of the app profile. The order of priority goes from lowest number to highest number. For example, a value of `-3` would be installed before an app profile with a higher number value. There are no upper and lower limits, and you may specify positive and negative integers.  The default value is `0`. ",
```
@SivaanandM I think this clarifies the concept and removes any questions the user might have.